tests/modules/images/webp-uploads/load-tests.php

```diff
@@ -602,6 +610,14 @@ public function it_should_use_smaller_jpg_image_when_smaller_size_preferred() {
 	 * @test
 	 */
 	public function it_should_replace_the_featured_image_to_web_p_when_requesting_the_featured_image() {
+		add_filter(
+			'webp_uploads_image_sizes_with_additional_mime_type_support',
+			function( $sizes ) {
+				$sizes['1536x1536'] = '1536x1536';
+				return $sizes;
+			}
+		);
```

Comment:
Same as above, let's remove this.

We should then either remove the assertion below that this doesn't include any `.jpeg`, or (maybe better), we should ensure that it only includes `.jpeg` for sizes that do not have additional MIME type support (e.g. for the above `1536x1536`).

Reply:
Yes, removed and updated this test to ensure that the `src` attribute contains the webp version of the image which satisfy the requirements for this test - we confirm that the correct webp version has been used for the post thumbnail image. Does it look good to you?

Reply:
LGTM.